include/lsst/meas/algorithms/CoaddPsf.h

```diff
@@ -36,6 +37,18 @@
 
 namespace lsst { namespace meas { namespace algorithms {
 
+class CoaddPsfControl {
+public:
+
+    LSST_CONTROL_FIELD(warpingKernelName, std::string,
+                       "Name of warping kernel; choices: lanczos3,lanczos4,lanczos5,bilinear,nearest");
+    LSST_CONTROL_FIELD(cacheSize, int, "Warping kernel cache size");
+
+    explicit CoaddPsfControl(std::string _warpingKernelName="lanczos3", int _cacheSize=10000) :
```

Comment:
I don't believe we have an established convention for this, but in cases like this I tend to use a trailing underscore instead of a leading one.  Not a big deal either way.